### ISSUE TITLE
Chat scrolled to end resists to scroll when using a trackpad

### ISSUE THREAD

**Issue by Joehuu** [ppy/osu#13614]
**Describe the bug:** I'm using a MacBook trackpad and it refuses to scroll up unless I move my fingers fast on the trackpad.

**Screenshots or videos showing encountered issue:** 

https://user-images.githubusercontent.com/35318437/122880037-65637b80-d2ee-11eb-9204-7605636d28fa.mp4

**osu!lazer version:** 2021.619.1

**Comment by peppy** [comment]
Just to be clear, you're expecting it to hold your position as long as your fingers are on the trackpad?

This will require framework changes I believe. Also, another discrepancy is that when letting go with a trackpad velocity/inertia is not correctly applied in line with expectations.

**Comment by Joehuu** [comment]
Not sure what that means, but it's a chat-only issue with the `ScrollToEnd` update logic.

And yeah the velocity part has been bugging me for awhile.

**Comment by peppy** [comment]
Hard to explain, but I don't think I'm experiencing the same "locking" behaviour that you're pointing out. It may just be an implementation difference between bluetooth and built-in trackpads:

https://user-images.githubusercontent.com/191335/122886430-ebc59f00-d37a-11eb-9806-c4a6823b8b37.mp4

**Comment by Joehuu** [comment]
We may be getting our scroll directions wrong so here's another video in a test scene.

https://user-images.githubusercontent.com/35318437/122888003-4832ab00-d2f6-11eb-912f-61c667e0ae17.mp4

**Comment by peppy** [comment]
Yeah, you're describing the same thing. I guess you're just scrolling slow enough for it to look like a potentially different issue (which was fixed in the past).

I think this could be renamed to "scrolling on a trackpad doesn't maintain position until fingers are released". Either way, this requires some very large changes as we need to hook on to API that we aren't yet, specific to apple devices, where you can tell when the fingers have been lifted from a scroll operation.
